lib/yard/docstring_parser.rb

```diff
@@ -281,18 +281,33 @@ def self.after_parse_callbacks
       seen_names = []
       infile_info = "\n    in file `#{parser.object.file}' " \
                     "near line #{parser.object.line}"
+      splat_param = parser.object.parameters.any? { |name, _| name =~ /\A\*/ }
+      param_tags = []
       parser.tags.each do |tag|
-        next if tag.is_a?(Tags::RefTagList) # we don't handle this yet
         next unless tag.tag_name == "param"
+        if tag.is_a?(Tags::RefTagList)
+          if tag.name
+            param_tags << tag
+          elsif !(CodeObjects::Proxy === tag.owner)
+            param_tags += tag.tags
+          end
+          # if the ref tag doesn't have a name and doesn't yet resolve, do nothing
+        else
+          param_tags << tag
+        end
+      end
```

Comment:
You can flatten down this `if` and improve readability IMO by using `.filter_map` to iterate here. Consider:
```ruby
param_tags = parser.tags.filter_map do |tag|
  next unless tag.tag_name == "param"
  next tag if !tag.is_a?(Tags::RefTagList) || tag.name
  
  tag.tags unless CodeObjects::Proxy === tag.owner
end
```

`filter_map` is Ruby 2.7+ and I'm not sure about Yard's current minimum Ruby version, so if it's lower you can use `map{ ... }.compact` instead.